### DISCUSSION TITLE
Change default data path and move volumes

### DISCUSSION THREAD

**Discussion by gboor** [longhorn/longhorn#3855] [Q&A]
I have a 3 node bare-metal RKE cluster using Longhorn for PVs. I recently added a new disk to all 3 nodes and mounted it under /srv. The longhorn volumes are currently under /var/lib/longhorn, but I would like to move them to /srv/longhorn.

In the UI, I can update the default data path, but I am not clear on what will happen when I do. Will longhorn move the volumes from /var/lib/longhorn to /srv/longhorn, or will it create new volumes there only, or will all volumes be recreated?

Ideally, I would like to move existing data from 1 location to the other with minimal disruption. Is this possible?

**Comment by derekbit** [comment]
You can add the `/srv/longhorn` to your node, and the evict the disk `/var/lib/longhorn`. Then, the replicas will be moved to the new disk.

**Reply by gboor** [reply]
How do you mean "evict the disk"? /var/lib is part of the root disk, so I cannot unmount it...

I did try changing the default data path in the config, but it appears that only applies to new nodes and not to existing ones. Deleting a replica of a volume recreated it in the old /var/lib/longhorn dir...

**Reply by gboor** [reply]
Nevermind, I figured it out!

Under nodes I edited the nodes one by one. Set the old disk to unschedulable and request eviction, added the new disk in the new folder. All volumes were moved, then I deleted the old disks.

Thanks for pointing me in the right direction!
